modules/simple_forms_api/app/services/simple_forms_api/pdf_filler.rb

```diff
@@ -14,10 +14,14 @@ def initialize(form_number:, data:)
     def generate
       template_form_path = "#{TEMPLATE_BASE}/#{form_number}.pdf"
       generated_form_path = "tmp/#{form_number}-tmp.pdf"
```

Comment:
Is this being used anywhere now or can it be removed?

Reply:
good question. yep! it is still being used. 

before the flow was: 
start: template_form_path
end: generated_form_path
then stamp it
- this procedure has a strange bug where the stamp sometimes is a separate editable layer in the pdf

new flow is:
start: template_form_path
copy to: stamped_form_path and stamp it
end: generated_form_path
- now our form will just be stamped already before going to the generated_form_path step, which seems to solve the issue

Reply:
oh, my bad! yeah, it see it being used. Thanks for the explanation!